ee/tabby-webserver/src/lib.rs

```diff
@@ -40,14 +42,32 @@ pub async fn attach_webserver(
     let ctx = Arc::new(ServerContext::new(conn, logger, code));
     let schema = Arc::new(create_schema());
 
+    repositories::load_dataset().await.unwrap();
+
+    let repo_routes = Router::new()
+        .route(
+            "/:name/resolve/",
```

Comment:
The axum [doc](https://docs.rs/axum/latest/axum/struct.Router.html#wildcards) says `wildcard path doesn’t match empty segments`, so this path will fallback to `ui::handler` if not specify explicitly here.

Reply:
Let's create vendor mime type: `application/vnd.directory+json`